docs/products/redis/concepts/ssl-connectivity.rst

```diff
@@ -0,0 +1,85 @@
+SSL connectivity
+================
+
+Client support for SSL-encrypted connections
+--------------------------------------------
+
+Default support
+~~~~~~~~~~~~~~~
+Aiven for Redis uses SSL encrypted connections by default. This is shown by the use of ``rediss://`` (with double s) prefix in the ``Service URI``. 
+
+.. Tip::
+    You can find the ``Service URI`` on `Aiven console <https://console.aiven.io/>`_.
+
+Since **Redis 6**, the redis-cli tool itself supports SSL connections; therefore, you can connect directly to your service using::
+
+    redis-cli -u rediss://username:password@host:port
+
+Alternatively, you can use the third-party `redli tool <https://github.com/IBM-Cloud/redli>`_::
+
+    redli -u rediss://username:password@host:port
+
+
+Not every Redis client supports SSL-encrypted connections.
+In such cases, you would have to turn off SSL to use these clients, which is allowed but not recommended. In those cases, you have some options:
+
+
+Set up stunnel process
+~~~~~~~~~~~~~~~~~~~~~~
+
+One workaround for this, it is to set up a `stunnel process <https://www.stunnel.org/index.html>`_ on the client side to handle encryption for the clients that do not support SSL connections. 
+
+You can use the following stunnel configuration, for example ``stunnel.conf``, to set up a stunnel process.
+::
+
+    client = yes
+    foreground = yes
+    debug = info
+    delay = yes
+
+    [redis]
+    accept = 127.0.0.1:6380
+    connect = myredis.testproject.aivencloud.com:28173
+    TIMEOUTclose = 0
+    ; For old services only. New ones use Let's Encrypt and there's no
+    ; CA cert available from Aiven console. Most environments trust
+    ; Let's Encrypt by default without any explicit CAfile config.
+    ; CAfile = /path/to/optional/project/cacert/that/you/can/download/from/aiven/console
+
+To understand the global options of the stunnel configuration, please check `Stunnel Global Options <https://www.stunnel.org/static/stunnel.html#GLOBAL-OPTIONS>`_.
+
+For ``service-level option``, the following parameters are configured:  
+
+``accept`` => *[host:]port*
+  **Accept connections on specified address**
+
+
+
+``connect`` => *[host:]port*
+  **Connect to a remote address** 
+
+
+
+``TIMEOUTclose`` => *seconds*
+  **Time to wait for close_notify**
+
+.. note:: It is important to make changes accordingly to your service. On the *Overview* page you can find your **Overview** > **Host** and **Overview** > **Port** to configure the ``connect`` parameter.
+
+Note that when SSL is in use we have a separate service terminating the SSL connections before they are forwarded to Redis. This process has a connection timeout of its own independent of Redis' connection timeout. If you allow very long Redis timeouts, this frontend service may end up closing the connection before the Redis timeout has expired. By the time of writing this, timeout is set to 12 hours.
```

Comment:
This paragraph is not clear in my opinion, where are the two timeouts? Can we explain it better?

Reply:
This was not clear for me as well, but I could clarify and now, it is. 

https://github.com/aiven/devportal/pull/429/commits/96c3eaae5088032a2f01aa93878d27e82e43c222

Thanks for pointing this so we could make it more clear. What do you think now?